### PR TITLE
0.10 Release Notes 

### PR DESCRIPTION
**Pull request by nkakkar81**
Added Release Notes for 0.10. The following additions have been done:
1. New Features
2. Deprecated features
3. Fixed Issues
No known issues have been added for this release.

### CODE REVIEW

File: modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+[id="openshift-pipelines-release-notes"]
+= Release Notes for OpenShift Pipelines 0.10 Developer Preview
+:context: openshift-pipelines-release-notes-0.10
+
+OpenShift Pipelines provides a cloud-native CI/CD experience based on the Tekton project. It has the following salient features:
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc:5]: I think the use of the word 'salient' is not needed. 

'It has the following features:' would suffice

File: modules/ROOT/pages/release_notes.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+* Standard Kubernetes-native pipeline definitions (CRDs)
+* Serverless pipelines with no CI server management overhead
+* Extensibility to build images using any K8s tool, such as S2I, Buildah, JIB, and Kaniko.
+* Portability across all Kubernetes distribution
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: distributions*

File: modules/ROOT/pages/release_notes.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
+[id="openshift-pipelines-release-notes"]
+= Release Notes for OpenShift Pipelines 0.10 Developer Preview
+:context: openshift-pipelines-release-notes-0.10
+
+OpenShift Pipelines provides a cloud-native CI/CD experience based on the Tekton project. It has the following salient features:
+
+* Standard Kubernetes-native pipeline definitions (CRDs)
+* Serverless pipelines with no CI server management overhead
+* Extensibility to build images using any K8s tool, such as S2I, Buildah, JIB, and Kaniko.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: or* Kaniko

File: modules/ROOT/pages/release_notes.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+== Getting support
+
+[discrete]
+=== Product
+To report a bug, or to suggest new features and enhancements, file an issue in the link:https://github.com/openshift/tektoncd-pipeline-operator[GitHub Project]. Include as many details as possible in the description of the issue.
+
+
+[discrete]
+=== Documentation
+If you find an error or have suggestions for improving the documentation, file an issue in link:https://github.com/openshift/pipelines-docs[pipelines documentation] repository.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: I think the term 'report' might work better than 'file'

File: modules/ROOT/pages/release_notes.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+For questions and feedback subscribe to the link:mailto:pipelines-interest@redhat.com[pipelines-interest@redhat.com]  mailing list or join the link:https://tektoncd.slack.com/archives/CKWCQD2EP[#openshift-pipelines] channel on Tekton Slack.
+
+== Deprecated Features
+
+The following items are deprecated in 0.8.0, and will be removed in the next release:
+
+* `EventListenerTrigger.interceptor` is now deprecated. Use Interceptors instead.
+* `EventListenerTrigger.binding` is now deprecated. Use `bindings` instead.
+* `status` field is now removed from `Pipeline` and `PipelineResource`. This field will stay in `v1alpha1`, however it will be removed in the next API version.
+* `.status.podName` field is now deprecated and should not be used for finding the pod associated with TaskRun. Use label selectors to identify Tekton owned pods.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: Back ticks? ----  `TaskRun`

File: modules/ROOT/pages/release_notes.adoc [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+[discrete]
+=== Getting in touch with the product team
+For questions and feedback subscribe to the link:mailto:pipelines-interest@redhat.com[pipelines-interest@redhat.com]  mailing list or join the link:https://tektoncd.slack.com/archives/CKWCQD2EP[#openshift-pipelines] channel on Tekton Slack.
+
+== Deprecated Features
+
+The following items are deprecated in 0.8.0, and will be removed in the next release:
+
+* `EventListenerTrigger.interceptor` is now deprecated. Use Interceptors instead.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: I think 'has been deprecated'  over 'is now deprecated', but this is just a personal preference.

File: modules/ROOT/pages/release_notes.adoc [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+* `EventListenerTrigger.binding` is now deprecated. Use `bindings` instead.
+* `status` field is now removed from `Pipeline` and `PipelineResource`. This field will stay in `v1alpha1`, however it will be removed in the next API version.
+* `.status.podName` field is now deprecated and should not be used for finding the pod associated with TaskRun. Use label selectors to identify Tekton owned pods.
+
+
+== Fixed Issues
+=== Redhat Fixed Issues
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: Red Hat*

File: modules/ROOT/pages/release_notes.adoc [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://issues.redhat.com/browse/SRVKP-429[SRVKP-429]: Tekton Pipelines can now be deployed on an OpenShift cluster running in Google Cloud Platform (GCP).
+* link:https://issues.redhat.com/browse/SRVKP-549[SRVKP-549]: Previously, the Pipelines section used to continuously load on the catalog/source-to-image page. This issue is now solved by adding checks for permissions.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/release_notes.adoc]: This issue is now resolved...

**Review by nebelsprachen — COMMENTED** [review]
Just some minor suggestions :)

File: modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc:5]: I think Salient makes sense as Tekton Pipeline may have a lot of features, and  here we are listing the main/important features.

File: modules/ROOT/pages/release_notes.adoc (hunk 2 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: Accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 3 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: Accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 4 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 5 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 6 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: Would prefer 'is now deprecated' :)

File: modules/ROOT/pages/release_notes.adoc (hunk 8 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+* Serverless pipelines with no CI server management overhead
+* Extensibility to build images using any K8s tool, such as S2I, Buildah, JIB, or Kaniko.
+* Portability across all Kubernetes distributions
+* Powerful CLI for interacting with pipelines
+* Improved user experience through tight integration with OpenShift Developer Console
+
+For an in-depth walk-through of OpenShift Pipelines, follow this link:https://github.com/openshift/pipelines-tutorial/[tutorial] to build your first cloud-native pipeline. Visit  link:https://openshift.github.io/pipelines-docs/docs/0.10.5/index.html[OpenShift Pipelines documentation] to learn more about OpenShift Pipelines.
+
+== Notable Features
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 0.10:
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: S/what's/the new features?

File: modules/ROOT/pages/release_notes.adoc [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+
+* Triggers v 0.2.1:  Support added for multiple `triggerbindings`, user configurable `interceptors`, `jsonpath` in `triggerbindings`, labeling generated resources and dynamic client for custom resources in `eventlistener`.
+
+* Pipeline Templates: Adds pipeline templates which will be available to a user for reuse through the *+Add* flows in the Developer perspective of the OpenShift web console.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: s/which will be/ that are

File: modules/ROOT/pages/release_notes.adoc [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+* Script Enhancements: Multiple enhancements made to the script mode when defining task steps:
+** Task steps in script mode no longer need to specify a shebang line. Scripts that do not start with shebang, by default, will have a value of `#!/bin/sh`
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: the script mode L25 and L26

File: modules/ROOT/pages/release_notes.adoc [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+** Support for arguments in script mode
+
+** Support for script mode for Conditions
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: Support in the script mode?

File: modules/ROOT/pages/release_notes.adoc [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+
+** Script mode now inserts `set -xe` to scripts that don’t include a shebang `#!`.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: the scripts...do not
Do not use contractions.

File: modules/ROOT/pages/release_notes.adoc [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+
+* Support for workspace in `Task`, `TaskRun`, `Pipeline` and `PipelineRun` to simplify sharing files between tasks in a pipeline
+
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc:33]: Either add periods or remove them but make it consistent.

File: modules/ROOT/pages/release_notes.adoc [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+[discrete]
+=== Getting in touch with the product team
+For questions and feedback subscribe to the link:mailto:pipelines-interest@redhat.com[pipelines-interest@redhat.com]  mailing list or join the link:https://tektoncd.slack.com/archives/CKWCQD2EP[#openshift-pipelines] channel on Tekton Slack.
+
+== Deprecated Features
+
+The following items are deprecated in 0.8.0, and will be removed in the next release:
+
+* `EventListenerTrigger.interceptor` is now deprecated. Use Interceptors instead.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: backticks `interceptors` or `Interceptors` ?

File: modules/ROOT/pages/release_notes.adoc [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+* `EventListenerTrigger.binding` is now deprecated. Use `bindings` instead.
+* `status` field is now removed from `Pipeline` and `PipelineResource`. This field will stay in `v1alpha1`, however it will be removed in the next API version.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: If it is going to stay for `v1alpha1` and removed later, should we be listing it here already? Also should this be V1alpha1 with no back ticks?

File: modules/ROOT/pages/release_notes.adoc [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+* `.status.podName` field is now deprecated and should not be used for finding the pod associated with `TaskRun`. Use label selectors to identify Tekton owned pods.
+
+
+== Fixed Issues
+=== Red Hat Fixed Issues
+* link:https://issues.redhat.com/browse/SRVKP-429[SRVKP-429]: Tekton Pipelines can now be deployed on an OpenShift cluster running in Google Cloud Platform (GCP).
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: in/on

File: modules/ROOT/pages/release_notes.adoc [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://issues.redhat.com/browse/SRVKP-549[SRVKP-549]: Previously, the Pipelines section used to continuously load on the catalog/source-to-image page. This issue is now resolved by adding checks for permissions.
+
+
+=== Tekton Fixed Issues
+* link:https://github.com/tektoncd/pipeline/pull/1756[#1756]: Previously, an error was observed when a pipeline was run on a service account with a long secret name. With this bug fix, the length of volumes generated from secret names is now restricted.
+* link:https://github.com/tektoncd/pipeline/pull/1675[#1675]: In the previous release, any resolved digests would get appended to the input image name, which resulted in invalid duplicate digests when the input was specified by digest. This change uses `ref.context()`, which omits the digest and appends the resolved digest to the end of the image name.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: the digest

File: modules/ROOT/pages/release_notes.adoc [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://github.com/tektoncd/pipeline/pull/1688[#1688]: Before this change, sidecars that are injected into the Pod by external Mutating Admission Controllers would not be guaranteed to be ready before starting the first step, and would not be correctly stopped when steps finish. With this fix, pipelines look for any non-step container which indicates the presence of a sidecar container, when waiting for or stopping sidecars.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: Too many 'to be' 
Suggest...would not always be ready before the first step started and would not correctly stop when the steps finished.

File: modules/ROOT/pages/release_notes.adoc [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://github.com/tektoncd/pipeline/pull/1693[#1693]: Previously, PullRequest PipelineResource was not reading the `scm.PR.Head.Sha` correctly, causing it to fail when adding a comment on the PR. This fix sets the  `scm.PR.Sha` to `scm.PR.Head.Sha` when loading from disk.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: ...read the `scm.PR.Head.Sha` incorrectly
the disk

File: modules/ROOT/pages/release_notes.adoc [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://github.com/tektoncd/pipeline/pull/1812[#1812]: Previously, a task was marked as successful even when it was OOMKilled by the controller. With this fix, the task is marked as failed.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: `OOMKilled` ?

File: modules/ROOT/pages/release_notes.adoc [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://github.com/tektoncd/pipeline/pull/1813[#1813]: A fix is added to correctly pass the `-submodules` flag to the `git-init` module.
+* link:https://github.com/tektoncd/pipeline/pull/1722[#1722]: Previously, the start time for all the steps in a task was reported to be the same as the start time for the first step. This has now changed to report the actual start time of the step, in accordance with the defined sequence in which the steps are executed in a task.
+* link:https://github.com/tektoncd/pipeline/pull/1863[#1863]: Previously, Gitlab status upload used to fail with missing commit error when using `pipelineresource` of type `pullrequest`. With this bug fix, the data is preloaded into the `resource.PullRequest` used in the `FromDisk` method.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: ...a missing...

File: modules/ROOT/pages/release_notes.adoc [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://github.com/tektoncd/pipeline/pull/1907[#1907]: Previously, the task used to get stuck in the `pending` mode for error `CreateContainerConfigError`. With this fix, the task is set to a `failed` state whenever  `CreateContainerConfigError` error is encountered.
```
Comment (Preeticp) [inline modules/ROOT/pages/release_notes.adoc]: extra space after whenever

File: modules/ROOT/pages/release_notes.adoc (hunk 12 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: modified to - Support for the script mode in Conditions

File: modules/ROOT/pages/release_notes.adoc (hunk 21 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: That was the terminology used in the bug. Not sure what it means. Have modified it to "killed" for now.

File: modules/ROOT/pages/release_notes.adoc (hunk 9 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 11 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 12 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 13 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 19 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 20 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 22 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

File: modules/ROOT/pages/release_notes.adoc (hunk 23 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: done

**Comment by nkakkar81** [comment]
@Preeticp - Comments are now incorporated

File: modules/ROOT/pages/release_notes.adoc (hunk 21 above)
Comment (Preeticp) [reply modules/ROOT/pages/release_notes.adoc]: I meant, should it be under backticks?

File: modules/ROOT/pages/release_notes.adoc (hunk 16 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: have changed to V1alpha1

File: modules/ROOT/pages/release_notes.adoc (hunk 21 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: ah ok..should be changed back to `OOMKilled`

File: modules/ROOT/pages/release_notes.adoc (hunk 15 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted - change to `interceptors`

File: modules/ROOT/pages/release_notes.adoc (hunk 17 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 18 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 14 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc:33]: accepted, put period at the end

File: modules/ROOT/pages/release_notes.adoc (hunk 16 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted - change to V1alpha1

File: modules/ROOT/pages/release_notes.adoc (hunk 17 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: in

File: modules/ROOT/pages/release_notes.adoc (hunk 18 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: accepted

File: modules/ROOT/pages/release_notes.adoc (hunk 15 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc]: `interceptors`

File: modules/ROOT/pages/release_notes.adoc (hunk 14 above)
Comment (nkakkar81) [reply modules/ROOT/pages/release_notes.adoc:33]: add periods

**Pull request merged**